cloudbuild/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# This Dockerfile builds the image used in Cloud Build CI to run 'make test'.
+# To push a new image, run 'gcloud builds submit --project=cloud-eng-council --tag gcr.io/cloud-eng-council/make .'
+# from this directory.
+
+FROM debian
+
+RUN apt-get update && apt-get install -y build-essential python python-pip
+RUN pip install yapf
```

Comment:
Should we use Python 3?

Reply:
yapf supports python 2 and 3 per the doc. I don't know if we need to install both for it to work.